src/pydap/responses/dmr.py

```diff
@@ -0,0 +1,113 @@
+"""The DMR response.
```

Comment:
Builds a xml representation of the data, so that pydap can serve a dataset containing arrays and groups. Does not yet work with Sequences and Structures. That will be another PR